arkouda/alignment.py

```diff
@@ -363,10 +365,14 @@ def search_intervals(vals, intervals):
             matchintervalidx = broadcast(segs, hitidx, matches.size)
             # make sure not to include any of the bounds themselves
             validmatch = (matches >= low.size) & (matches < boundary)
-            # indices of unique values found (translated from concat keys)
-            uvalidx = matches[validmatch] - low.size
+            matchintervalidx = matchintervalidx[validmatch]
+            # indices of values found (translated from concat keys)
+            validx = matches[validmatch] - low.size
```

Comment:
could we rename these to use `_` while we're in here. Most of them are fairly easy to parse out but my brain gets confused especially on `matchintervalidx`. I know it's meant to be read `match_interval_idx` but sometimes i read it as `match_inter_valid_x` because of `valid_match` right beside it

I just realized it could also be `match_inter_val_idx`. I never actually read it like that, but I could see someone getting it mixed up that way since `val_idx` is also pretty close by

Reply:
I am in agreement with @pierce314159 on this.